src/test/java/JUnitTest.java

```diff
@@ -0,0 +1,160 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
A J-unit test should be created corresponding to the class which need to be tested. You may try to navigate to the class > right click > add test... to add a test for the class